packages/esm-patient-chart-app/src/visit/visits-widget/visit-detail-overview.component.tsx

```diff
@@ -30,6 +32,11 @@ function VisitDetailOverviewComponent({ patientUuid }: VisitOverviewComponentPro
     <div className={styles.tabs}>
       <Tabs>
         <TabList aria-label="Visit detail tabs" contained>
+          {showCurrentVisitTab && (
+            <Tab className={styles.tab} id="visit-summaries-tab">
+              {t('currentVisit', 'Current visit')}
```

Comment:
I wanted use one config to hide the current/active visit tab and to also show all forms as opposed to the general encounters tab. do you think it would be important to seperate the two ?

Reply:
I could rename the tab to `Active visit` to avoid any confusion with the current visit tile